edge/pkg/eventbus/mqtt/filter.go

```diff
@@ -0,0 +1,158 @@
+package mqtt
```

Comment:
we should add some unit tests for the filter.go

Reply:
> we should add some unit tests for the filter.go

done with unit test